services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movie(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids is None and actors_ids is None:
+        return queryset
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(genres_id__in=genres_ids, actors_id__in=actors_ids)

```

Comment:
There is an issue with the filtering logic in `get_movie`. The filter should use `genres__id__in=genres_ids` and `actors__id__in=actors_ids` instead of `genres_id__in=genres_ids` and `actors_id__in=actors_ids`. This is because `genres` and `actors` are ManyToMany fields, and you need to specify the related field's `id` for filtering.